documentation/staging/content/userguide/base-images/_index.md

```diff
@@ -1,19 +1,21 @@
 ---
```

Comment:
I have some general questions for @ddsharpe  about updating the WDT install - the directions in this chapter only cover updating JRE & WLS - but not WDT. If there's no time pressure, I recommend fixing that. IMO, the chapter should specifically discuss how to download WDT, put it in teh WIT cache, and tell WIT to include the updated WDT install in the rebase.  Is there time pressure @mriccell ?

Reply:
And what are the incantations for doing this @ddsharpe ?